api/src/main/java/io/grpc/MethodDescriptor.java

```diff
@@ -262,6 +264,17 @@ public String getServiceName() {
     return serviceName;
   }
 
+  /**
+   * A convenience method for {@code extractMethodName(getFullMethodName())}.
+   *
+   * @since 1.32.0
+   */
+  @Nullable
+  @ExperimentalApi("https://github.com/grpc/grpc-java/issues/5635")
+  public String getMethodName() {
```

Comment:
We should add something to the name to make users think whether they want the full name or this one. "simple" or "bare" or similar. Ditto for extract method.

I'm not sure if we have any such term today, as method name is normally fully qualified. Maybe another language has such a thing?

Reply:
I am not aware of another language having solved this properly. The naming confusion is very real. Generally the full(y qualified) method name gets its own name (also here: `getFullMethodName()`) and the other one is just `MethodName`. But from the options I picked Bare for now. Let me know what you think.